pkg/fanal/test/integration/registry_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build integration
-// +build integration
-
```

Comment:
It is integration test. Don't need to remove this.

Reply:
if I understand correctly - if you return this, then broken test will be fixed.

Reply:
I restored this, but the same test is failing when I run `mage test:unit`. Actually, it's failing for me (locally) even on the `main` branch.

Reply:
It should help with broken test
```suggestion
			runtimes, err := WithRuntimes(types.AllRuntimes)
			assert.NoError(t, err)
			img, cleanup, err := NewContainerImage(context.Background(), tt.args.imageName, tt.args.option, runtimes)
```

Reply:
Unit tests are passing now, thanks!